CafeTime/CafeTime/SignUpView.swift

```diff
@@ -378,10 +420,16 @@ class SignUpView: UIView, UIGestureRecognizerDelegate, UITextFieldDelegate {
         UITextField.cleanFields([passwordTextField,emailTextField,nameTextField,foodTypeTextField,numberOfTablesTextField, countryTextField])
     }
     
+    
     // MARK: TextField Delegate
     
     func textField(_ textField: UITextField, shouldChangeCharactersIn range: NSRange, replacementString string: String) -> Bool {
-        let invalidCharacters = CharacterSet(charactersIn: "0123456789").inverted
-        return string.rangeOfCharacter(from: invalidCharacters, options: [], range: string.startIndex ..< string.endIndex) == nil
+        if textField == numberOfTablesTextField {
+            let invalidCharacters = CharacterSet(charactersIn: "0123456789").inverted
```

Comment:
you can move this validation method to Utils files. probably we can use it somewhere else later in the project. and email validation method can be placed there.